loadtest/user/userentity/websocket.go

```diff
@@ -154,19 +149,16 @@ func (ue *UserEntity) listen(errChan chan error) {
 					chanClosed = true
 					break
 				}
-				client.UserTyping(msg.channelId, msg.parentId)
-			case <-client.PingTimeoutChannel:
-				timedOut = true
+				if err := client.UserTyping(msg.channelId, msg.parentId); err != nil {
+					errChan <- fmt.Errorf("userentity: error in client.UserTyping: %w", err)
+				}
 			}
-			if chanClosed || timedOut {
+			if chanClosed {
```

Comment:
I'm 0/5 on this but we can use named loops for this one if it is non-idiomatic.

Reply:
Just to clarify, you want a label at the start of the outer loop, and use a "break label" to directly break to the outer loop ? But that will still put you at the start of the loop, not end of it. And then you'd have to check some variable again. Unless I am missing something, or you meant something else ?

Reply:
resolved offline.